app/models/answer.rb

```diff
@@ -265,7 +265,7 @@ def replicate_location_values
     if location_type_with_value?
       tokens = self.value.split(" ")
       LOCATION_ATTRIBS.each_with_index do |a, i|
-        self[a] = tokens[i] ? BigDecimal.new(tokens[i]) : nil
+        self[a] = tokens[i] ? (BigDecimal(tokens[i]) rescue nil) : nil
```

Comment:
This form of `rescue` is discouraged I think. Can you move the decimal coercion to a supporting private method, use a named exception, and add a comment explaining why we need to do this?